organization/src/integrationTest/kotlin/com/cosmotech/organization/service/OrganizationServiceIntegrationTest.kt

```diff
@@ -1283,1447 +1283,1502 @@ class OrganizationServiceIntegrationTest : CsmRedisTestBase() {
-          assertNotEquals(defaultRoleCreated, defaultRoleUpdated)
-        }
+    @Test
+    fun `getOrganizationAccessControl with owned organization and current user`() {
+      assertDoesNotThrow {
+        val name = "o-connector-test-1"
+        val organizationRegistered =
+            organizationApiService.registerOrganization(createTestOrganization(name))
+        val organizationRole =
+            organizationApiService.getOrganizationAccessControl(
+                organizationRegistered.id!!, TEST_ADMIN_USER_ID)
+        assertNotNull(organizationRole)
+        assertEquals(ROLE_ADMIN, organizationRole.role)
+        assertEquals(TEST_ADMIN_USER_ID, organizationRole.id)
       }
+    }
 
-      @Test
-      fun `setOrganizationDefaultSecurity with not owned organization and no WRITE_SECURITY_PERMISSION`() {
-        assertDoesNotThrow {
-          val name = "o-connector-test-1"
-          runAsDifferentOrganizationUser()
-          val organizationRegistered =
-              organizationApiService.registerOrganization(
-                  createTestOrganizationWithSimpleSecurity(
-                      name, TEST_ADMIN_USER_ID, ROLE_NONE, ROLE_NONE))
-          val defaultRoleCreated = organizationRegistered.security?.default
-          runAsPlatformAdmin()
-          assertNotNull(
-              organizationApiService.setOrganizationDefaultSecurity(
-                  organizationRegistered.id!!, OrganizationRole(ROLE_ADMIN)))
-          val defaultRoleUpdated =
-              organizationApiService.getOrganizationSecurity(organizationRegistered.id!!).default
-          assertNotEquals(defaultRoleCreated, defaultRoleUpdated)
-        }
+    @Test
+    fun `getOrganizationAccessControl with owned organization and non-existing user`() {
+      assertThrows<CsmResourceNotFoundException> {
+        val name = "o-connector-test-1"
+        val organizationRegistered =
+            organizationApiService.registerOrganization(createTestOrganization(name))
+        organizationApiService.getOrganizationAccessControl(organizationRegistered.id!!, "UNKOWN")
       }
+    }
 
-      @Test
-      fun `getOrganizationAccessControl with owned organization and current user`() {
-        assertDoesNotThrow {
-          val name = "o-connector-test-1"
-          val organizationRegistered =
-              organizationApiService.registerOrganization(createTestOrganization(name))
-          val organizationRole =
-              organizationApiService.getOrganizationAccessControl(
-                  organizationRegistered.id!!, TEST_ADMIN_USER_ID)
-          assertNotNull(organizationRole)
-          assertEquals(ROLE_ADMIN, organizationRole.role)
-          assertEquals(TEST_ADMIN_USER_ID, organizationRole.id)
-        }
+    @Test
+    fun `getOrganizationAccessControl with not owned organization, READ_SECURITY permission and existing user`() {
+      assertDoesNotThrow {
```

Comment:
assertDoesNotThrow should be specific else it's the same as not having it

Reply:
same comment as above